source/NVDAObjects/IAccessible/winword.py

```diff
@@ -47,12 +47,6 @@ def _get_ignoreEditorRevisions(self):
 	ignoreFormatting=False
 
 	def event_caret(self):

```

Comment:
I think we need to research why this code was   ever added. It should be possible to do this with a git blame.

Reply:
I tent to agree with @leonardder that checking with git blame in which commit this was introduced is safer. Would you be able to do this?